### DISCUSSION TITLE
[NZP 2015 COMP] [RELEASE] Cavern

### DISCUSSION THREAD

**Discussion by MotoLegacy** [nzp-team/nzportable#162] [Map Releases]
## This is a manual migration of forum user Naievil's map release `Cavern`. Below contains the original post content. You can view the original post [here](https://nzportable.forumotion.com/t2072-nzp-2015-comp-release-cavern).

Hey everyone,

This map is called Cavern, a hole somewhere in the western US that is used as a sort of mine. This map features All of the basic perks with the exception of Flopper since it is essentially useless. The map does not feature a mystery box. The only good thing about this map are the visuals, which are pretty great. Please note that huge lagspikes will occur on PSP and PC, though it is playable. This was tested on a PSP 1000, and it may not occur on later models.

This map was tested using the 1.02 Unofficial Patch, as most users are not using 1.03 due to the fact that we cannot pause on PSP

Screenshots:
![shot_0](https://user-images.githubusercontent.com/26030250/216126626-5962c6b3-c00d-4a57-b4e1-94e008e3ab40.jpg)
![shot_1](https://user-images.githubusercontent.com/26030250/216126624-7f20ddd1-c464-4682-ac0e-2b0b3a406d79.jpg)
![shot_2](https://user-images.githubusercontent.com/26030250/216126623-75cecb28-0187-4da6-8bd1-d9d3b52502b0.jpg)
![shot_3](https://user-images.githubusercontent.com/26030250/216126620-44bd548f-fb79-4e1b-b813-81ad1e14f110.jpg)
![shot_4](https://user-images.githubusercontent.com/26030250/216126619-4299b173-2c81-4c72-9d08-71ac45c20ee7.jpg)
![shot_5](https://user-images.githubusercontent.com/26030250/216126618-77f2ce57-5716-49da-a69a-01860591ac21.jpg)
![shot_6](https://user-images.githubusercontent.com/26030250/216126617-d637e77b-86cc-4552-a2c9-ddaa41f54b15.jpg)
![shot_7](https://user-images.githubusercontent.com/26030250/216126616-1ebd0551-d6b3-452d-a990-2f99802a7a70.jpg)
![shot_8](https://user-images.githubusercontent.com/26030250/216126611-abaec4ab-25aa-4499-a0bf-d85861b54fc9.jpg)
![shot_9](https://user-images.githubusercontent.com/26030250/216126609-f8fc2164-0afb-4890-9d94-8a3ff3e98dfb.jpg)
![shot_91](https://user-images.githubusercontent.com/26030250/216126606-ea00cb60-be67-40bf-99df-6861e0702f56.jpg)
![shot_92](https://user-images.githubusercontent.com/26030250/216126605-d5413137-8e62-4d00-9679-64bb6f12cc59.jpg)
![shot_93](https://user-images.githubusercontent.com/26030250/216126604-d43ef067-a5be-4c49-8494-9c9c43d7ace3.jpg)
![shot_94](https://user-images.githubusercontent.com/26030250/216126600-6cc806f3-1b31-45af-b5a5-3ed9d2bf847e.jpg)
![shot_95](https://user-images.githubusercontent.com/26030250/216126598-ef4d4351-4c9f-44b6-bd53-2f4abf9acb44.jpg)

Nothing more can be done to this map, sadly. I wish everyone luck in their entries.

This map lags like you would not believe. Do NOT ask me to fix this map.

If you can decompile it and make it run smooth, go for it. You have my permission.

# [DOWNLOAD](https://archive.org/download/nzp_cavern/nzp_cavern.zip)

**Comment by Paindeluxe** [comment]
I‘ll try to decompile and optimize it (;

_It was a failed attempt_
